python/cuml/test/test_fil.py

```diff
@@ -35,18 +37,21 @@
     import xgboost as xgb
 
 
-def simulate_data(m, n, k=2, random_state=None, classification=True,
-                  bias=0.0):
+def simulate_data(m, n, k=2, n_informative='auto', random_state=None,
+                  classification=True, bias=0.0):
+    if n_informative == 'auto':
+        n_informative = n // 5
     if classification:
         features, labels = make_classification(n_samples=m,
                                                n_features=n,
-                                               n_informative=int(n/5),
+                                               n_informative=n_informative,
+                                               n_redundant=n - n_informative,
```

Comment:
Not sure whether this is exactly the same, but I'll defer it to the Python code owners.